### ISSUE TITLE
Login error

### ISSUE THREAD

**Issue by biluses** [evanstoddard/Ajax-Login#4]
When you register and try to login it always give you the "There was an unknown error. Please try again later." ERROR.

if you try to access by URL to home.php, it creates automatically a PHPSESSID and redirect you to index.html again.

Where is the problem with the login if it gives you a CORRECT in data_handling.js on line 17 as checked in Console?

thank you

**Comment by evanstoddard** [comment]
Yeah this project is about 3 years old or so.  I haven't touched it since.  I would look at one of the forks or a better login solution.  This one isn't very good.

Cheers.

**Comment by biluses** [comment]
any recomendation for this?

**Comment by biluses** [comment]
if($.trim(response) === 'Correct') and it works now for the check_login function, i thnk that's a good Ajax-login even if it's 3years old.

**Comment by evanstoddard** [comment]
It is fixed now.  There was an unmerged pull request.  Make sure you make changes to your config file and set up the sql database.  This isn't a very secure system.  I will eventually revamp it and rewrite a lot of it but if you're working on something simple and not too secretive then this will work as a basic login.
